lib/tasks/whitehall_importer.rb

```diff
@@ -121,5 +129,45 @@ def embed_contacts(body, contacts)
         "[Contact:#{embed}]"
       end
     end
+
+    def primary_publishing_organisation(organisations)
+      unless organisations
+        raise AbortImportError, "Must have at least one organisation"
+      end
+
+      primary_publishing_organisations = organisations.select do |organisation|
+        organisation["lead"]
+      end
+
+      unless primary_publishing_organisations.any?
+        raise AbortImportError, "Lead organisation missing"
+      end
+
+      if primary_publishing_organisations.count > 1
+        raise AbortImportError, "Cannot have more than one lead organisation"
+      end
+
+      primary_publishing_organisation = primary_publishing_organisations.find do |organisation|
+        organisation["lead_ordering"] == 1
```

Comment:
Sorry to be a pain but I wonder if we definitively know that the list will be ordered starting from 1?

I think it might be slightly better to do 
`primary_organisations = organisations.select { |o| o["lead"] }.sort { |o| o["lead_ordering"] }` and then we can use first item there.

Reply:
Good point.  Have updated the PR.